.env.example

```diff
@@ -72,3 +72,11 @@ GOOGLE_CSE_ID=
 # Miscellaneous options
 # Skip loading Chroma.
 OVERWRITE_CHROMA=true
+
+# Enable SentenceEmbedding model served via BentoML
+# For local embedding service, use:
+#   docker run --rm -p 3000:3000 ghcr.io/bentoml/sentence-embedding-bento:0.1.0
```

Comment:
docker run --rm -p 3000:3000 ghcr.io/bentoml/sentence-embedding-bento:0.1.0
Unable to find image 'ghcr.io/bentoml/sentence-embedding-bento:0.1.0' locally
docker: Error response from daemon: Head "https://ghcr.io/v2/bentoml/sentence-embedding-bento/manifests/0.1.0": unauthorized.

any idea?

Reply:
Let me publish the docker image, I think chaoyu forgot to add it 😄

Reply:
I published the image bug forgot to make it public, it should be available now!